Makefile

```diff
@@ -193,6 +193,9 @@ errdoc: install-go-tools
 	@echo "generator errors.toml"
 	./scripts/check-errdoc.sh
 
+docker-build-test:
+	docker build --no-cache -t tikv/pd .
```

Comment:
Thanks, maybe you can check whether docker is installed. Some developers may not have docker locally.

Reply:
what should be the exit code when docker is not installed ?

Reply:
what about print the message and exit with a non-zero code

Reply:
@Yisaer @nolouch how about now?